### PR TITLE
refactor(packaging): use authoring for package editing

### PR DESCRIPTION
**Pull request by petrjasek**
it switches to package edit within authoring, but from outside
there is single edit/view action for any type of item.

SD-3156

### CODE REVIEW

File: client/tasks/options/watch.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ module.exports = {
     },
     assets: {
         options: {livereload: true},
-        tasks: ['ngtemplates'],
+        tasks: [''],
```
Comment (sivakuna-aap) [inline client/tasks/options/watch.js]: With this change, does it build template.js in local when there is a change in html?

Comment (petrjasek) [inline client/tasks/options/watch.js]: no, my bad, this should be part of different PR

**Comment by ioanpocol** [comment]
:+1:

**Comment by amagdas** [comment]
:+1:
